manim/scene/streaming_scene.py

```diff
@@ -0,0 +1,111 @@
+from .. import config
+from ..mobject.frame import FullScreenRectangle as Frame
+from ..renderer.stream_renderer import StreamCairoRenderer
+from ..utils.simple_functions import get_parameters
+from .scene import Scene
+
+
+class Stream:
+    """This class is really intended for inheritance of the style:
+
+    >>> class Streamer(Stream, Scene): # doctest: +SKIP
+    ...     pass
+    ...
+    >>>
+
+    This order is paramount. This :class:`Stream` class does the switch to
+    the specialized renderer, which uses the :class:`StreamFileWriter` to
+    handle specialized streaming services. That explains the calls to super,
+    which dig through the MRO of a class instead of using just a single
+    implementation contained in Scene. Okay, the references in super probably
+    point to things in the :class:`Scene` class only, but it's already happened so...
+
+    PS: You can probably already tell using this class on its own will bring you
+    errors.
+
+    """
+
+    def __init__(self, **kwargs):
+        camera_class = self.mint_camera_class()
+        renderer = StreamCairoRenderer(camera_class=camera_class)
+        super().__init__(renderer=renderer, **kwargs)
+        # To identify the frame in a black background
+        self.add(Frame())
+        self.setup()
+
+    @classmethod
+    def mint_camera_class(cls):
+        """Only __init__ arguments have the camera class now. In order for the
+        specialized renderer to be used, the scene's camera class must be found.
```

Comment:
This is not a very good one-line summary of this method, I don't really get an idea of what this does.
Also_ `__init__` needs to be set as code.

```suggestion
        """Only ``__init__`` arguments have the camera class now. In order for the
        specialized renderer to be used, the scene's camera class must be found.
```

Reply:
Note to self, update this